[2025-02-14T22:42:51.361+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_ETL.get_location_dict manual__2025-02-14T22:42:46.653678+00:00 [queued]>
[2025-02-14T22:42:51.373+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_ETL.get_location_dict manual__2025-02-14T22:42:46.653678+00:00 [queued]>
[2025-02-14T22:42:51.374+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 1
[2025-02-14T22:42:51.390+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): get_location_dict> on 2025-02-14 22:42:46.653678+00:00
[2025-02-14T22:42:51.394+0000] {standard_task_runner.py:57} INFO - Started process 217 to run task
[2025-02-14T22:42:51.402+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data_ETL', 'get_location_dict', 'manual__2025-02-14T22:42:46.653678+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/weather_data_dag.py', '--cfg-path', '/tmp/tmp2m3un8ms']
[2025-02-14T22:42:51.405+0000] {standard_task_runner.py:85} INFO - Job 42: Subtask get_location_dict
[2025-02-14T22:42:51.534+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data_ETL.get_location_dict manual__2025-02-14T22:42:46.653678+00:00 [running]> on host 2079ceed134c
[2025-02-14T22:42:51.765+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airscholar' AIRFLOW_CTX_DAG_ID='weather_data_ETL' AIRFLOW_CTX_TASK_ID='get_location_dict' AIRFLOW_CTX_EXECUTION_DATE='2025-02-14T22:42:46.653678+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-14T22:42:46.653678+00:00'
[2025-02-14T22:42:51.783+0000] {logging_mixin.py:149} INFO - {'39.8765,-105.7658': 1, '39.4937,-106.1594': 2, '39.9375,-105.584': 3, '40.4662,-106.7809': 4, '39.1911,-120.2557': 5, '45.2931,-111.3629': 6, '39.2567,-120.1334': 7, '46.9348,-121.4838': 8, '43.5946,-110.8461': 9, '39.4789,-106.0784': 10, '39.6198,-106.3696': 11, '43.11,-72.9103': 12, '50.0735,-122.9594': 13, '45.9698,6.879': 14}
[2025-02-14T22:42:51.836+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-02-14T22:42:51.868+0000] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=weather_data_ETL, task_id=get_location_dict, execution_date=20250214T224246, start_date=20250214T224251, end_date=20250214T224251
[2025-02-14T22:42:51.897+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-02-14T22:42:51.944+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
